gen/Dockerfile.ejs

```
@@ -70,7 +70,7 @@ RUN git clone --depth=1 https://github.com/novnc/websockify.git /root/websockify
 
 
 ## Import UPM.
-FROM replco/upm:light@sha256:9a07f4b0912f29692436be2a28c7024fee1d93eb16c3d3eed9fe6309dd121b37 AS upm
+FROM replco/upm:light@sha256:e05689b5b0d51a2316f8a87b1a9d6cbf90d98b19a424dbb924ee3d0b1cc17bfc AS upm
```

```suggestion
FROM replco/upm:light@sha256:fc729728ec975f918953ad6a12497de712ca0858344ddbbeb638f692c60c6167 AS upm
```
I think we want this sha instead. From: https://hub.docker.com/layers/replco/upm/light/images/sha256-fc729728ec975f918953ad6a12497de712ca0858344ddbbeb638f692c60c6167?context=explore
In the `DIGEST` under the name of the image